optlang/symbolics.py

```diff
@@ -84,12 +86,14 @@ def __getnewargs__(self):
     def add(*args):
         if len(args) == 1:
             args = args[0]
-        return sum(args)
+        return Add(*args)
```

Comment:
Out of curisosity, which cases require Add instead of sum?

Reply:
The empty list :) At least for cobrapy it is important that Object.expression is always an expression. Otherwise it may become an int (sum([]) = int(0)) which breaks a lot of stuff in optlang. For instance Constraint.variables will explode if all coefficients are zero.